.github/workflows/tag_build_and_deploy.yml

```diff
@@ -0,0 +1,96 @@
+name: Tag Build and Deploy
+on:
+  push:
+    tags: 
+      - 'v[0-9]+(\.[0-9]+)*([-\.]\S+)?'
+
+env: 
+  DOMAIN: jellyroom.membrane.work
+  JELLYFISH1_ADDRESS: jellyfish.membrane.work
+  JELLYFISH2_ADDRESS: jellyfish1.membrane.work
+  BACKEND_ADDRESS: server.jellyroom.membrane.work
+  PROJECT: staging
+  COMPOSE_FILE_NAME: docker-compose-deploy.yaml
+  TARGET_DIR: jellyfish_videoroom
+
+jobs:
+  deploy1:
+    runs-on: ubuntu-latest
+    steps:  
+      - uses: actions/checkout@v2
+
+      - name: Prepare .env file for the deployment
+        id: variables_population
+        run: |
+          echo "DOMAIN=${{env.DOMAIN}} 
+          JELLYFISH1_IP=${{secrets.STAGING_HOST}}
+          JELLYFISH2_IP=${{secrets.STAGING2_HOST}}
+          SERVER_API_TOKEN=${{secrets.SERVER_API_TOKEN}}
+          SECRET_KEY_BASE=${{secrets.SECRET_KEYBASE}}
+          JELLYFISH1_ADDRESS=${{env.JELLYFISH_ADDRESS}}
+          JELLYFISH2_ADDRESS=${{env.JELLYFISH2_ADDRESS}}
+          JELLYFISH_ADDRESSES=${{env.JELLYFISH1_ADDRESS}} ${{env.JELLYFISH2_ADDRESS}}
```

Comment:
Isn't it a typo that we have `JELLYFISH_ADDRESS` and `JELLYFISH1_ADDRESS`?

Reply:
yep, typo. fixed